apps/public/lib/getOccupancyDescription.tsx

```diff
@@ -0,0 +1,15 @@
+import t from "@bloom/ui-components/src/helpers/translator"
+import { Listing } from "@bloom/ui-components/src/types"
+
+const getOccupancyDescription = (listing: Listing) => {
```

Comment:
My only question here is whether this should be a function with "get" in the name here, or whether it can just be a subcomponent that renders in?

Reply:
I'm not sure what you mean here. It's just returning a translated string…so not really rendering anything per se.